### ISSUE TITLE
[c++2py] Should the wrapping really be tied to comments of the type ///  ?

### ISSUE THREAD

**Issue by Wentzell** [TRIQS/triqs#468]
When I wrap the following test class 

```c++2py
class test{ 
  /*
   * The constructor
   *
   * @param j Some integer
   */
  test(int j); 
}; 
```

I get the desc file 

```python
# Generated automatically using the command :
# c++2py.py test.hpp -p -m test -o test
from wrap_generator import *

# The module
module = module_(full_name = "test", doc = "", app_name = "test")

# All the triqs C++/Python modules

# Add here all includes beyond what is automatically included by the triqs modules
module.add_include("test.hpp")

# Add here anything to add in the C++ code at the start, e.g. namespace using
module.add_preamble("""
""")

# The class test
c = class_(
        py_type = "Test",  # name of the python class
        c_type = "test",   # name of the C++ class
        doc = r"",   # doc of the C++ class
)

module.add_class(c)

module.generate_code()
```
i.e. the constructor is not properly picked up. This seems to be linked to the type of the comment 
in front of it, as it works with ```///```. Should we really tie the wrapping to a certain type of comment?

**Comment by parcollet** [comment]
By default, libclang ignores the //, and picks only the ///, /** (oxygen like comments).
We need to add -fparse-all-comments to the compiler options to change this and pick up
all comments.

**Comment by parcollet** [comment]
Solved in 1.6 dev branch.
